packages/server-core/src/media/storageprovider/s3.storage.ts

```diff
@@ -12,18 +12,32 @@ import {
 } from './storageprovider.interface'
 
 export class S3Provider implements StorageProviderInterface {
-  bucket = config.aws.s3.staticResourceBucket
-  cacheDomain = config.aws.cloudfront.domain
-  provider: AWS.S3 = new AWS.S3({
-    accessKeyId: config.aws.keys.accessKeyId,
-    secretAccessKey: config.aws.keys.secretAccessKey,
-    region: config.aws.s3.region,
-    s3ForcePathStyle: true
-  })
+  bucket = config.server.storageProvider === 'aws' ? config.aws.s3.staticResourceBucket : config.ipfs.fleekKeys.bucket
```

Comment:
we dont need to change any of this, you just need to supply the alternative configuration variables in the .env

Reply:
@HexaField explicit variables will not create any confusion in the future otherwise dev-ops needs to be very careful when deploying to project 

do you still want me to change it?